requirements-test.txt

```diff
@@ -1,24 +1,33 @@
 certifi==2016.8.31 \
-#    --hash=sha256:f7708a42d86f29ccc7c8c4ff9d34a8d854d8d78eb2973d1f28406bb43d6b8919
+    --hash=sha256:172844a3e2896c924e6aa2657b2e56586a724f27edd0e9592be9353f3a3f6c73
```

Comment:
Yes, Even I was surprised, I hope I'm doing it the right way. So this is what I did:

- Uncomment the hashes in requirements.txt file
- pip uninstall -r requirements.txt
- pip install -r requirements.txt
    - I get errors in this step where the hash isn't matching, and the expected hash is displayed by pip. So, I replace the given hash with the suggested hash. See the logs http://pastebin.com/Pcac94Ff

Also, If I'm right in detecting the right hash values, then we should think about implementing hash checking in balrog in development phase because with hash values changing, this can result in frequent failures while deploying.